WearStandaloneGoogleSignIn/app/src/main/res/values/strings.xml

```diff
@@ -19,4 +19,5 @@
     <string name="wear_signout_button_text">Sign out</string>
     <string name="google_signin_successful">Google Sign-In successful!</string>
     <string name="signout_successful">Successfully signed out!</string>
+    <string name="server_client_id">TODO_ADD_CLIENT_ID_XXXXXXX.apps.googleusercontent.com</string>
```

Comment:
You added this string but you're not using it anywhere, correct?

Reply:
Correct, well, I'm not using it directly, but the google sign in API uses it.

Reply:
I added a comment to call that out.